assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/index.test.js

```diff
@@ -275,6 +311,109 @@ describe( 'AudienceSelectionPanel', () => {
 		} );
 	} );
 
+	describe( 'ErrorNotice', () => {
+		it( 'should not display an error notice when there are no errors', async () => {
+			const { container, waitForRegistry } = render(
+				<AudienceSelectionPanel />,
+				{
+					registry,
+				}
+			);
+
+			await waitForRegistry();
+
+			expect( container ).not.toHaveTextContent( 'Data loading failed' );
+			expect( container ).not.toHaveTextContent(
+				'Insufficient permissions, contact your administrator.'
+			);
+		} );
+
+		it.each( [
+			[ 'resyncing available audiences', 'syncAvailableAudiences', [] ],
+			[ 'retrieving user count', 'getReport', [ reportOptions ] ],
+		] )(
+			'should display an error notice when there is an insufficient permissions error while %s',
+			async ( _, fn, args ) => {
+				const error = {
+					code: 'test_error',
+					message: 'Error message.',
+					data: { reason: ERROR_REASON_INSUFFICIENT_PERMISSIONS },
+				};
+
+				provideUserInfo( registry, {
+					id: 1,
+					email: 'admin@example.com',
+					name: 'admin',
+					picture: 'https://path/to/image',
+				} );
```

Comment:
The options are unnecessary. The test is passing without it.
```suggestion
				provideUserInfo( registry );
```